spyder/widgets/explorer.py

```diff
@@ -1201,16 +1202,9 @@ def __init__(self, parent=None, name_filters=['*.py', '*.pyw'],
             widget.setIconSize(QSize(16, 16))
 
         # Layouts
-        blayout = QHBoxLayout()
-        blayout.addWidget(button_previous)
-        blayout.addWidget(button_next)
-        blayout.addWidget(button_parent)
-        blayout.addStretch()
-        blayout.addWidget(self.button_menu)
-
-        layout = QVBoxLayout()
-        layout.addLayout(blayout)
-        layout.addWidget(self.treewidget)
+        controls = [button_previous, button_next, button_parent, 'stretch',

```

Comment:
@timhoffm if you really want to go in that direction then we should have a `StretchWidget` or something but mixing text with widgets is just wrong and error prone cause you can have a typo and we are not even checking if it text or not, just checking if it is a specific text (not suggesting we even do that).